### PR TITLE
add simple launcher menu entry

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: src/cmd/rio/menu.c [hunk 1]
```diff
@@ -205,15 +238,9 @@ spawn(ScreenInfo *s)
 			signal(SIGINT, SIG_DFL);
 			signal(SIGTERM, SIG_DFL);
 			signal(SIGHUP, SIG_DFL);
-			if(termprog != NULL){
```
Comment (katcipis) [inline src/cmd/rio/menu.c:208]: Nice idea to extract this logic from here and inject a function =D

**Review by katcipis — APPROVED** [review]
LGTM

File: src/cmd/rio/menu.c [hunk 2]
```diff
@@ -80,6 +82,34 @@ Menu	egg =
 	version
 };
 
+void
+runterm() {
+	if(termprog != NULL){
+		execl(shell, shell, "-c", termprog, (char*)0);
```
Comment (katcipis) [inline src/cmd/rio/menu.c:88]: Isn't the "-c" option specific to some shells ? (I remember it vaguely from bash)

Comment (i4ki) [reply src/cmd/rio/menu.c:88]: -c is specific to all posix compliant shells...

http://pubs.opengroup.org/onlinepubs/009695399/utilities/sh.html

AFAIK, the shells below have this option:
sh, rc, bash, csh, dash, fish, zsh, nash

Comment (i4ki) [reply src/cmd/rio/menu.c:88]: I think that if you recite the sequence of shell names above very fast you get brain damage kkkk

Comment (katcipis) [reply src/cmd/rio/menu.c:88]: auheauheauheuaheuhauheuahe

Did not know that, thanks =)

**Pull request merged**